lib/generators.js

```diff
@@ -256,14 +253,15 @@ function _generatePaths(generated_routes, tags, customBlueprintMaps) {
                 tags: tag ? [tag.name] : [identity],
                 produces: ['application/json'],
                 consumes: ['application/json'],
-                parameters: parameter.concat(_getQueryParamsFromRouteQuery(route.query)).concat(_getBodyParamsFromRouteBody(route.body)),
+                parameters: parameter.concat(_getQueryParamsFromParameters(route.parameters))
```

Comment:
no need for this, since the query is doing same

Reply:
But 'query' is by default false..we are no able to add any query param which is required.

Reply:
```
 'get /users/me': {
        controller: 'UserController',
            action: 'login',
            swagger: {
           		summary: 'Authentication',
                description: 'This is for authentication of any user',
            query: [{ 
username: { type: 'string', required: true } 
}]
        }
```
under **query**